llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -1141,6 +1141,15 @@ static int getInstructionIDWithAttrMask(uint16_t *instructionID,
   return 0;
 }
 
+static bool isCCMPOrCTEST(InternalInstruction *insn) {
+  return (insn->opcodeType == MAP4) &&
+         (((insn->opcode & 0xfe) == 0x38) || ((insn->opcode & 0xfe) == 0x3a) ||
```

Comment:
Add a variable for `insn->opcode & 0xf`

Reply:
Done in another way. I use `switch-case` to avoid the duplication.